### ISSUE TITLE
Using Less-Than-Sign from Drag and Drop area?

### ISSUE THREAD

**Issue by raybaranoski** [dthies/tinymce4-mathslate#2]
Hi, Daniel...  I looked as best I could for a solution to this in either a config or a settings file, but couldn't, so I figured I would ask here.  I apologize if this is not the correct place to post this.

This error appears in yellow and red when dragging a less-than-sign into the window.  

"Error parsing MathML: error on line 1 at column 96: StartTag: invalid element name"

The symbol is accepted and it appears to work, but is there a way to suppress this or escape the < symbol? or html entity? 

Any help would be appreciated.  Thank you for the work done on this project.

**Comment by dthies** [comment]
Hi  Ray,

Thanks for reporting this this is the right place or on on of the other integrations. I have not noticed this before, but was able to reproduce it. It is a bug in the configuration file.  I just tried the configuration editor at http://dthies.github.io/moodle-editor_tinymce-mathslate/configuration.html and it came up with an improvement. In config.js replace          _["mo",{},"<"],_ with           _["mrow",{"tex":["<"]},[["mo",{},"&lt;"]]],_ and that should fix the problem.

**Comment by dthies** [comment]
An additional problem is that mouse over of < and > does not display the characters correctly but instead the encoded entities &amp;lt; and &amp;gt;.

**Comment by raybaranoski** [comment]
Thank you very much, testing now...

**Comment by raybaranoski** [comment]
Hi, sorry for the delay, but was able to look into this more as it initially did not work for me.  I couldn't get the online configuration tool to respond positively, so I just worked directly in my environment.  Is it ok to use <em>["mrow", {"tex":["<"]},[["mo",{},"&#60;"]]],</em> using the html decimal entity in the config.json,  as this worked for me?  Thanks again for your work on this... If you don't see an issue with using that value for <, then I'll be ok with this config option.

**Comment by dthies** [comment]
Yes, I think that should be correct. I may have had that and mistakenly have let gitbub editor decode it.
